src/runtime_src/core/tools/common/TestRunner.h

```diff
@@ -39,10 +36,16 @@ class TestRunner : public JSONConfigurable {
              boost::property_tree::ptree& _ptTest);
     void logger(boost::property_tree::ptree& ptree, const std::string& tag, const std::string& msg);
     bool search_and_program_xclbin(const std::shared_ptr<xrt_core::device>& dev, boost::property_tree::ptree& ptTest);
-    std::string findPlatformPath(const std::shared_ptr<xrt_core::device>& _dev,
-                  boost::property_tree::ptree& _ptTest);
+    std::string findPlatformPath(const std::shared_ptr<xrt_core::device>& dev,
+                                 boost::property_tree::ptree& ptTest);
+    std::vector<std::string> findPlatformPaths(const std::shared_ptr<xrt_core::device>& dev,
+                                               boost::property_tree::ptree& ptTest);
```

Comment:
This can be a private function

Reply:
It cannot. There are tests that call it such as `TestBandwidthKernel`.